src/components/sidekick/styles.scss

```diff
@@ -23,7 +23,7 @@
   }
 
   &__tab-content {
-    width: 100%;
+    width: inherit;
```

Comment:
@ratik21, If the content inside the .sidekick__tab-content container is not wide enough to fill the available space, using width: 100% could result in the container being narrower than its parent container (270px in this case). This is because width: 100% makes the container expand to occupy the full width of its parent, but not more than the available content width.

On the other hand, using width: inherit ensures that the tab content container always inherits the width of its parent container, regardless of the content width. So even if the content inside the container is narrower than the parent container's width (270px), the tab content container will still be set to the parent's width of 270px.

@dalefukami would you agree?